coutoire/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coutoire",
-  "version": "1.3.5",
+  "version": "1.3.6",
```

Comment:
Somehow this is out of sync with the other version numbers. I guess we just fix it manually...